src/backend/distributed/worker/worker_partition_protocol.c

```diff
@@ -614,6 +615,26 @@ CitusCreateDirectory(StringInfo directoryName)
 }
 
 
+#ifdef WIN32
+static bool
+FileIsLink(char *filename, struct stat filestat)
+{
+	DWORD attributes = GetFileAttributes(filename);
+	return attributes & FILE_ATTRIBUTE_REPARSE_POINT;
```

Comment:
Why not use `pgwin32_is_junction()` instead?

Reply:
Nice catch. How did you find it?

Reply:
I usually grep for functions/constants we use to check if postgres already uses them, and if yes, how :)